async/delay.ts

```diff
@@ -1,13 +1,16 @@
 // Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
 // This module is browser compatible.
+import { unrefTimer } from "../_deno_unstable.ts";
 
 export interface DelayOptions {
   signal?: AbortSignal;
+  /** Indicates whether the process can exit before the timer ends. This is `false` by default. */
+  persistent?: boolean;
```

Comment:
If we use `persistent`, then the meaning (effect) is the opposite of `unref`. The doc would be `Indicates whether the process should continue to run as long as timer existing` and it defaults to true.

Reply:
My misunderstanding. Fixed now!